rpcapi/accounthandlers.go

```
@@ -31,10 +33,44 @@ func (r *RPCAPI) getAccount(request *api.APIrequest) (*api.APIresponse, error) {
 	response.Nonce = new(uint32)
```

Regarding the previous lines (not part of this PR but related):

```
	// account does not exist
	if acc == nil {
		// response ok meaning the account does not exist
		return &response, nil
	}
```
I think we should return an error if the account does not exists.